cli_test/cli_posts_test.go

```diff
@@ -26,7 +26,7 @@ func TestDesmosCLIPostsCreateNoMediasNoPollData(t *testing.T) {
 
 	// Later usage variables
 	subspace := "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e"
-	message := "message"
+	message := "message#hashtag"
```

Comment:
I don't think this message is valid. It should probably be something like `"message #hashtag"`. Of course, we have the problem that CLI simulation tests do not allow for spaces in between the arguments :thinking: